app/Http/Controllers/DealExposeController.php

```
@@ -0,0 +1,639 @@
+            'entities' => $paginator->items(),
+            'current_page' => $paginator->currentPage(),
+            'last_page' => $paginator->lastPage(),
+            'per_page' => $paginator->perPage(),
+            'total' => $paginator->total(),
+        ]);
+    }
+
+    /**
+     * @return \Illuminate\Contracts\Pagination\LengthAwarePaginator<int, array<string, mixed>>
+     */
+    private function availableEntitiesPage(int $companyId, string $scope, string $search, ?int $projectId, int $perPage)
```

PHPStan (2.2.8)

[error] PHPDoc tag `@return` contains generic type Illuminate\Contracts\Pagination\LengthAwarePaginator<int, array<string, mixed>> but interface Illuminate\Contracts\Pagination\LengthAwarePaginator is not generic.


(generics.notGeneric)